src/test/java/io/github/binaryoverload/GeneralUtilsTest.java

```diff
@@ -52,16 +52,16 @@ public void checkStringLengthNegative() {
 
     @Test
     public void verifyPathPositive() {
-        GeneralUtils.verifyPath("cat.cat.cat", '.');
-        GeneralUtils.verifyPath("ca-t.cat.cat", '.');
-        GeneralUtils.verifyPath("cat_cat.cat", '.');
-        GeneralUtils.verifyPath("cat-cat-cat", '-');
+        GeneralUtils.verifyPath("cat.cat.cat", '.', new char[]{});
+        GeneralUtils.verifyPath("ca-t.cat.cat", '.', new char[]{'-'});
+        GeneralUtils.verifyPath("cat_cat.cat", '.', new char[]{'_'});
+        GeneralUtils.verifyPath("cat-cat-cat", '-', new char[]{});
     }
 
     @Test(expected = IllegalArgumentException.class)
     public void verifyPathNegative() {
-        GeneralUtils.verifyPath("cat.cat.cat..", '.');
-        GeneralUtils.verifyPath("cat-cat-cat-", '-');
+        GeneralUtils.verifyPath("cat.cat.cat..", '.', new char[]{});
+        GeneralUtils.verifyPath("cat-cat-cat-", '-', new char[]{});
```

Comment:
Possibly add another test to test for the specific case of unallowed characters?

Reply:
Sure.